src/main/scala/io/iohk/ethereum/App.scala

```diff
@@ -0,0 +1,32 @@
+package io.iohk.ethereum
+
+import java.net.{InetSocketAddress, URI}
+
+import akka.actor.ActorSystem
+import io.iohk.ethereum.crypto._
+import io.iohk.ethereum.network.{NodeInfo, ServerActor, PeerManagerActor}
+
+object App {
+
+  val nodeKey = generateKeyPair()
+
+  def main(args: Array[String]): Unit = {
+    val listenHostname = "127.0.0.1"
+    val listenPort = 9076
```

Comment:
I have been working on providing a list of peers to connect to from an `application.conf` file so I could easily insert this values in that configuration file.